src/models/msg/nft/msg_issue_denom.ts

```diff
@@ -0,0 +1,47 @@
+import * as R from 'ramda';
+import { Categories } from '../types';
+
+class MsgIssueDenom {
+  public category: Categories;
+  public type: string;
+  public json: any;
+  public id: string;
+  public name: string;
+  public creators: string[];
+  public splitShares: MsgCoin;
+  public royaltyShare: MsgCoin;
```

Comment:
I don't think the type is `MsgCoin`

```
// NOTE: never use new(Dec) or else we will panic unmarshalling into the
// nil embedded big.Int
type Dec struct {
	i *big.Int
}
```

---

We don't need to log the whole model. Just whatever the tx message needs